packages/databricks-vscode/src/configuration/ui/AuthTypeComponent.ts

```diff
@@ -41,10 +48,20 @@ export class AuthTypeComponent extends BaseComponent {
         }
 
         if (authProvider === undefined) {
-            const label = "Login to Databricks";
+            const host = await this.configModel.get("host");
+            if (host === undefined) {
+                return [];
```

Comment:
If the bundle doesn't have a host defined we show a "failed to initialize target" notification right now, and then end up in a broken state when you can't really switch target anymore or even trigger the error again. 

At least here it would make sense to show a message about the absent host and prompt users to open and update the dabs config

Reply:
Fixed this state as well. Now we show a message saying "host is misconfigured". And once user fixes it, changes are picked up and processed automatically.